nix/modules/flake/rust.nix

```
@@ -40,8 +41,11 @@
             || lib.hasSuffix "flake-schemas/flake.lock" path
             || lib.hasSuffix "addstringcontext/flake.nix" path
             || lib.hasSuffix "addstringcontext/flake.lock" path
+            || lib.hasSuffix "metadata/flake.nix" path
+            || lib.hasSuffix "metadata/flake.lock" path
           ;
         };
+      defaultCraneArgs = import "${inputs.self}/nix/envs" { inherit (config.rust-project) src; inherit (pkgs) cachix fetchFromGitHub lib; };
```

Better to follow `haskell-flake` convention:

```suggestion
      defaults.crate.crane.args = import "${inputs.self}/nix/envs" { inherit (config.rust-project) src; inherit (pkgs) cachix fetchFromGitHub lib; };
```

---

https://github.com/juspay/omnix/pull/430/commits/344f6dca79ef335f21860e0125778209fe3750e9